elements/rh-navigation-secondary/rh-navigation-secondary.ts

```diff
@@ -73,13 +73,16 @@ function focusableChildElements(parent: HTMLElement): NodeListOf<HTMLElement> {
 export class RhNavigationSecondary extends LitElement {
   static readonly styles = [styles];
 
-  /**
-   * Color palette darker | lighter (default: lighter)
-   */
-  @colorContextProvider()
-  @property({ reflect: true, attribute: 'color-palette' }) colorPalette: NavPalette = 'lighter';
+  private static instances = new Set<RhNavigationSecondary>();
 
-  @queryAssignedElements({ slot: 'nav' }) private _nav?: HTMLElement[];
+  static {
+    window.addEventListener('keyup', (event: KeyboardEvent) => {
```

Comment:
```suggestion
    globalThis.addEventListener('keyup', (event: KeyboardEvent) => {
```

see if this crashes ssr

Reply:
relevant to subnav as well. https://github.com/RedHat-UX/red-hat-design-system/blob/96b5df39d5d4cd25d80d9c51e0abe92f09a1a4c0/elements/rh-subnav/rh-subnav.ts#L44